exp/api/v1alpha3/azuremanagedmachinepool_conversion.go

```diff
@@ -43,6 +43,7 @@ func (src *AzureManagedMachinePool) ConvertTo(dstRaw conversion.Hub) error {
 	dst.Spec.MaxPods = restored.Spec.MaxPods
 	dst.Spec.OsDiskType = restored.Spec.OsDiskType
 	dst.Spec.NodeLabels = restored.Spec.NodeLabels
+	dst.Spec.EnableUltraSSD = restored.Spec.EnableUltraSSD
```

Comment:
I think you need this line for v1a4 as well, that will fix the failing test :)

Reply:
Thanks, added!